src/sonic-yang-models/tests/yang_model_tests/tests_config/portchannel.json

```diff
@@ -10,6 +10,7 @@
                         "lanes": "65",
                         "mtu": 9000,
                         "name": "Ethernet0",
+                        "tpid": "0x8100",
```

Comment:
We can avoid updating old tests, since it is not mandatory field. Just add it in new tests.

Reply:
I am actually leveraging this same test config file for the new attribute TPID testing.  Are you saying that I should generate a new test file just for TPID purpose?  since TPID is a new attribute for both port and portchannel, I thought of just expanding on existing test case  instead of generating a new file for just one TPID test...
There are no other tests needed other than ensure that the model can catch invalid TPID values.